integration-tests/.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this modifications related to Cypress?

Reply:
We have such file in the root of our repository. 
In Sirius Web it is also present in the `integration-tests` directory, I assume this is here to make sure settings are correct even if `integration-tests` is opened as a standalone project. Let me know if I should remove it

Reply:
Ok, I see. Let's keep it then.